components/Badge/Badge.css

```diff
@@ -0,0 +1,19 @@
+.Badge {
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  width: 240px;
+}
+
+.iconContainer {
+  max-width: 100px;
```

Comment:
let's just hard-code this as `width` instead of `max-width` here and drop `width: 100%`